### DISCUSSION TITLE
Serial communication issue

### DISCUSSION THREAD

**Discussion by martin-vl** [arnemauer/Ducobox-ESPEasy-Plugin#130] [Q&A] [answered]
Last week I received my Ventilation gateway v8.6. First I want to say that looks nice and a good design. 
I only have issue with connecting the serial connection. When I activate both RF and Serial plugins the serial data is not received any more and there is a time out. While if I disable the RF plugin the serial plugins works fine.

Situation when Serial communication works fine:
<img width="1334" alt="Screenshot 2023-01-08 at 18 20 07" src="https://user-images.githubusercontent.com/9927920/211209916-eb011288-9a9d-4b7e-abe5-2d89ab21a0ea.png">

Situation when Serial communication is not working:
<img width="1330" alt="Screenshot 2023-01-08 at 18 22 30" src="https://user-images.githubusercontent.com/9927920/211209996-97899d1d-009c-43fa-b733-75b0948ebf2f.png">

When I remove the RF log I get the following messages:
```
17413721: [P156] DUCO FanSpeed: start read, eventid:4
17413721: [P156] DUCO FanSpeed: Start reading fanspeed
17413729: [P153] DUCO BOX SENSOR: start read, eventid:5
17413729: [P153] DUCO BOX SENSOR: Serial port in use, set flag to read data later.
17413739: WD   : Uptime 290 ConnectFailures 0 FreeMem 17416 WiFiStatus 3 ESPeasy internal wifi status: Conn. IP Init

17416871: [P156] DUCO FanSpeed: Serial reading timeout
17416871: [P156] DUCO FanSpeed: Stop reading from serial
17416872: [P153] DUCO BOX SENSOR: start read, eventid:5
17416872: [P153] DUCO BOX SENSOR: Start readBoxSensors5

17420028: [P153] DUCO BOX SENSOR: Serial reading timeout
17420028: [P153] DUCO BOX SENSOR: Stop reading from serial
17420032: [P151] DUCO SER GW: Start disableCoreLogLevel

17423187: [P151] DUCO SER GW: Serial reading timeout
17423187: [P151] DUCO SER GW: Stop reading from serial

17444940: WD   : Uptime 291 ConnectFailures 0 FreeMem 18376 WiFiStatus 3 ESPeasy internal wifi status: Conn. IP Init
```

Any idea how I can solve this issue?

**Comment by arnemauer** [comment, marked as answer]
Hi,

You need to select 8.3 and lower. Is it working with that setting?

**Reply by martin-vl** [reply]
Both plugins I set now on 8.3 and lower. It seems to work now. Thanks.
<img width="760" alt="Screenshot 2023-01-08 at 19 49 42" src="https://user-images.githubusercontent.com/9927920/211213580-f772a8c2-11c5-4a89-bfd9-dd36c662859c.png">

It is still strange, because I have version 8.6, but oke.
Thanks for your fast response.
